kubernetes/linux/main.sh

```diff
@@ -263,6 +263,24 @@ if [[ ((! -e "/etc/config/kube.conf") && ("${CONTAINER_TYPE}" == "PrometheusSide
       fi
 fi
 
+#Parse the configmap to set the right environment variables for agent config.
+#Note > tomlparser-agent-config.rb has to be parsed first before td-agent-bit-conf-customizer.rb for fbit agent settings
+if [ "${CONTAINER_TYPE}" != "PrometheusSidecar" ]; then
```

Comment:
why did we shift this above?

Reply:
IGNORE_PROXY_SETTINGS being set in tomlparser-agent-config which required at line 311.